### PR TITLE
dom-event-delegation - Kristian Krog

### PR DESCRIPTION
**Pull request by krog-kristian**
delegation

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var taskList = document.querySelector('.task-list');
+
+taskList.addEventListener('click', taskCheck);
+
+function taskCheck(event) {
+  console.log('target:', event.target);
```
Comment (shawnkost) [inline dom-event-delegation/main.js:6]: You're missing a console log for the `event.target.tagName`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Kristian!

File: dom-event-delegation/main.js (hunk 1 above)
Comment (krog-kristian) [reply dom-event-delegation/main.js:6]: target-tag

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**